src/main/java/com/jcabi/github/mock/MkNotifications.java

```diff
@@ -51,8 +52,37 @@
     public Iterable<Notification> iterate() {
         throw new NotImplementedException("MkNotifications#iterate");
     }
+
+    /**
+     * Get a single notification.
+     * @param number Notification id
+     * @return Notification
+     * @see <a href="https://developer.github.com/v3/activity/notifications/#view-a-single-thread">View a single thread</a>
+     */
     @Override
     public Notification get(final int number) {
         throw new NotImplementedException("MkNotifications#get");
     }
+
+    /**
+     * Marks all notifications on this repository as read.
+     * @see <a href="https://developer.github.com/v3/activity/notifications/#mark-notifications-as-read-in-a-repository">Mark notifications as read in a repository</a>
+     * @todo Implement this method
+     */
+    @Override
+    public void markAsRead() {
+        // Will be implemented later
```

Comment:
add a single todo in class description, which methods have to be implemented and how (see a todo that is already there)